unicode-transforms.cabal

```diff
@@ -89,7 +89,7 @@ library
 
     -- We depend on a lot of internal modules in text. We keep the upper bound
     -- inclusive of the latest stable version.
-    , text >= 1.1.1 && <= 1.2.4.1
+    , text >=1.1.1 && <=1.2.5.0 || >=2.0 && <2.1
```

Comment:
Are there more releases likely in 1.2 series?

Reply:
I don't think so, but it is not impossible. E. g., if we discover that some patches are needed for GHC 9.0.2 or 9.2.2.